### ISSUE TITLE
Should we introduce a `osnadmin channel info` subcmd?

### ISSUE THREAD

**Issue by yeasy** [hyperledger/fabric#4890]
### Current Status

osnadmin support following cmds today: join, list and remove.
If we want to get detailed information on specific channel, we use the `list` cmd with specific channel id.
```
List channel information for an Ordering Service Node (OSN). If the channelID flag is set, more detailed information will be provided for that channel.
```

### Expected

Another better option is to let `list` only do list all channels work, and introduce a separate subcmd such as `info` to do the detailed info query on specific channel.

### Solution

Add a new subcmd of `info`.

### Please let us know if you plan to work on this.

Yes!

**Comment by yeasy** [comment]
@denyeart how do you think?

**Comment by Priyansurout** [comment]
@yeasy can i also work on this?

**Comment by yeasy** [comment]
> @yeasy can i also work on this?

Let's wait for @denyeart 's comment first.

**Comment by Priyansurout** [comment]
ok

**Comment by denyeart** [comment]
Yes, I agree, `osnadmin channel info --channelID=<channel_name>` is a good enhancement, we look forward to a contribution!

**Comment by Priyansurout** [comment]
@yeasy so can i work on this now
